doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -839,6 +841,59 @@ Therefore, the most popular method is to take previous aerosol simulation result
 
 The aerosol initial input currently read by GOCART is the same format as the UFSAtm initial input data format of "gfs_data_tile[1-6].nc" in :numref:`Table %s <GridICFiles>`, so the aerosol initial conditions should be combined with the meteorological initial conditions as one initial input file. There are many tools available for this purpose. Ufs-utils in the global workflow that supports UFS models always provides a solution for this.
 
+-------
+AQM (CMAQ)
+-------
```

Comment:
The leading and trailing -- lines need to be at least as long as the text, otherwise building throws an error
```suggestion
--------------
AQM (CMAQ)
--------------
```

Reply:
Interesting, I did not know. I will note this in the future. Thanks.

Reply:
You could also fix another error that always shows up (unrelated to your additions). 

There are two references to "here" (hyperlinks) in the Introduction. We need a second underscore at the end of the hyperlink to resolve the build error.

```
diff --git a/doc/UsersGuide/source/Introduction.rst b/doc/UsersGuide/source/Introduction.rst
index 192264e4..af200e5a 100644
--- a/doc/UsersGuide/source/Introduction.rst
+++ b/doc/UsersGuide/source/Introduction.rst
@@ -39,9 +39,9 @@ for those elements, are listed below:

 The UFS Weather Model is currently included in two UFS Application releases.  These UFS Apps also contain pre- and post-processing components, a comprehensive build system, and workflows for configuration and execution of the application.

-The UFS WM v2.0 is included as part of the UFS Short Range Weather App, and details can be found `here <https://ufs-srweather-app.readthedocs.io/en/ufs-v1.0.0>`_.
+The UFS WM v2.0 is included as part of the UFS Short Range Weather App, and details can be found `here <https://ufs-srweather-app.readthedocs.io/en/ufs-v1.0.0>`__.

-The UFS WM v1.1 and v1.0 is included as part of the UFS Medium Range Weather App, and details can be found `here <https://ufs-mrweather-app.readthedocs.io/en/ufs-v1.1.0>`_.
+The UFS WM v1.1 and v1.0 is included as part of the UFS Medium Range Weather App, and details can be found `here <https://ufs-mrweather-app.readthedocs.io/en/ufs-v1.1.0>`__.
```

Reply:
> Should the UFS WM v2.0 be using a v1.0 link?

Oh i see it's the SRW vs MRW. nvm.